nixos/modules/services/web-servers/apache-httpd/default.nix

```diff
@@ -501,6 +501,14 @@ in
         '';
       };
 
+      restartSec = mkOption {
+        type = types.str;
+        default = "100ms";
+        description = ''
+          Time to sleep before restarting a service.
+        '';
+      };
+
```

Comment:
`restartSec` is redundant, one can change serviceConfig.RestartSec directly for apache via systemd service.